src/Symfony/Component/Security/Http/Firewall/ContextListener.php

```diff
@@ -166,13 +166,13 @@ protected function refreshUser(TokenInterface $token)
 
                 return $token;
             } catch (UnsupportedUserException $e) {
-                // let's try the next user provider
+                continue;
```

Comment:
why drop the comment (which is accurate) and add a continue (which is useless) ?